websites/factories.py

```diff
@@ -24,7 +31,7 @@ class WebsiteStarterFactory(DjangoModelFactory):
     slug = factory.Sequence(lambda n: "starter-%x" % n)
     source = FuzzyChoice(STARTER_SOURCES)
     commit = factory.Faker("md5")
-    config = EXAMPLE_SITE_CONFIG
+    config = json.dumps(EXAMPLE_SITE_CONFIG)
```

Comment:
I think this should be just `EXAMPLE_SITE_CONFIG`. If you try to create a `WebsiteStarter` you can see that `config` contains a JSON string rather than the unserialized contents of it

Reply:
I think you're right. I'll change